package.json

```diff
@@ -12,7 +12,9 @@
     "d3-v4-grid": "2.0.1",
     "errorhandler": "1.5.0",
     "express": "4.16.0",
+    "fs": "0.0.1-security",
     "googleapis": "1.1.5",
+    "https": "^1.0.0",
```

Comment:
Per our conservative approach to handling library versions, should this one just be plain `1.0.0`?

Reply:
yes, changing it now